presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcMetadata.java

```diff
@@ -142,9 +142,9 @@ public JdbcTableHandle getTableHandle(ConnectorSession session, SchemaTableName
                         handle.getCatalogName(),
                         handle.getSchemaName(),
                         handle.getTableName(),
-                        Optional.of(newColumns),
                         handle.getConstraint(),
-                        handle.getLimit()),
+                        handle.getLimit(),
+                        Optional.of(newColumns)),
```

Comment:
Builder for `JdbcTableHandle` would free us from modifying call sites, but it would also make it very error-prone.
Having to actually modify all the call sites forces me to go thru them and update them (add a condition, pass-thru some field or do nothing).
I am strongly against adding a builder for this class.

Reply:
Sure, but it's very very tempting to have a `JdbcTableHandle.builderFrom(JdbcTableHandle handle)` method, which totally undermines any safety which we really need here.

For this reason, I am convinced not having a builder class is less error-prone that having one for handling `JdbcTableHandle` transformations.